fetch/h1-parsing/incomplete-response.window.js

```diff
@@ -0,0 +1,17 @@
+// This only tests headers without a single newline after them for now.
+// See https://github.com/whatwg/fetch/issues/472 for context.
+
+const statusLine = "HTTP/1.1 200 OKAYISH\n";
+
+[
+  "header: value\t",
+  "header: value ",
+  "header: value\f",
```

Comment:
"" and "header: value\n" seems like the most important ones to test here.

Also seems worth testing "HTTP/1.1 200 OKAYISH" alone (no LF)

Per other comment, I'm concerned about large (HTTP-only) breakage from browsers actually enforcing this.

Reply:
I explicitly don't want to test `header: value\n` here as we do have interoperable behavior on not returning a network error there. Maybe that should change, but let's tackle that separately.

Reply:
Are you sure?  I'm pretty sure Chrome doesn't allow that over HTTPS.

Reply:
I guess I should create separate HTTPS tests then...